Source/dead.h

```diff
@@ -5,16 +5,17 @@
  */
 #pragma once
 
+#include <array>
 #include <cstdint>
 
 #include "engine.h"
 
 namespace devilution {
 
-#define MAXDEAD 31
+static constexpr unsigned MAXDEAD = 31;

```

Comment:
We should not be using ALL_CAPS for these types of constants.

Reply:
```suggestion
static constexpr unsigned MaxDead = 31;
```

Reply:
one word a.k.a `define` isn't 133t enough in 2021. Instead we need
`static constexpr unsigned uint8_t signed halfsigned` :P

Reply:
Done.

@galaxyhaxz Heh, that's true. However,  if you want to keep everything in the devilution namespace, you have to ditch almost all of your macros.

Reply:
@vlolteanu Ah, I'm in the process of learning C++ so wasn't sure the exact reason for these changes. Thanks for clarifying :)